pkgs/games/freedroidrpg/drop-build-deps.patch

```diff
@@ -0,0 +1,13 @@
+Do not embed paths to build-only depends (-I...SDL2-dev and friends)
+into savefile lua comments.
+--- a/src/savestruct_internal.c
++++ b/src/savestruct_internal.c
+@@ -486,8 +486,6 @@ void save_game_data(struct auto_string *strout)
+ 	autostr_append(strout,
+ 		"SAVEGAME: %s %s %s;sizeof(tux_t)=%d;sizeof(enemy)=%d;sizeof(bullet)=%d;MAXBULLETS=%d\n",
+ 		SAVEGAME_VERSION, SAVEGAME_REVISION, VERSION, (int)sizeof(tux_t), (int)sizeof(enemy), (int)sizeof(bullet), (int)MAXBULLETS);
+-	autostr_append(strout, "BUILD_CFLAGS: %s\n", BUILD_CFLAGS);
+-	autostr_append(strout, "BUILD_LDFLAGS: %s\n", BUILD_LDFLAGS);
```

Comment:
```suggestion
-	autostr_append(strout, "BUILD_CFLAGS: %s\n", BUILD_CFLAGS);
-	autostr_append(strout, "BUILD_LDFLAGS: %s\n", BUILD_LDFLAGS);
+	autostr_append(strout, "BUILD_CFLAGS: <ommited>\n");
+	autostr_append(strout, "BUILD_LDFLAGS: <ommited>\n");
```
this maybe creates less confusion when searching for those strings

Reply:
Sounds good. I used `"<hidden>"` wording.